yatsm/cli/pixel.py

```diff
@@ -190,15 +191,27 @@ def pixel(ctx, config, px, py, band, plot, ylim, style, cmap,
         )
 
 
-def plot_TS(dates, y):
+def plot_TS(dates, y, seasons):
     """ Create a standard timeseries plot
 
     Args:
         dates (iterable): sequence of datetime
         y (np.ndarray): variable to plot
     """
     # Plot data
-    plt.scatter(dates, y, c='k', marker='o', edgecolors='none', s=35)
+    if seasons == []:
+        plt.scatter(dates, y, c='k', marker='o', edgecolors='none', s=35)
+    else:
+        for date in dates:
+            if date.month == 11 or date.month == 12 or date.month == 1 or date.month == 2 or date.month == 3:
```

Comment:
Thanks for the feedback! I implemented the more general format you suggested and moved to the main `pixel` function. 

In switching to a list of months, I did have an issue with having October as the only month in the `FALL_MONTHS` list (i.e. `argument of type 'int' is not iterable`). Since September could also be considered part of this season based on the figure I shared, I made `FALL_MONTHS = (9, 10)` and fixed the description in the text to match. 

Also, the way the code is now, even if  `--seasons` is not enabled, the season variables are still being defined and passed to the `plot_TS` function. Not sure if this is bad practice. Let me know if there's a better way.